### PR TITLE
adjust the toggleItems code and remove toggleItems api

### PR DESCRIPTION
**Pull request by knoxHuang**
Re: cocos-creator/fireball#

Changes proposed in this pull request:
 * 调整 toggleItems 代码和移除 toggleItems api

@cocos-creator/engine-admins

### CODE REVIEW

**Review by jareguo — CHANGES_REQUESTED** [review]
123

File: cocos2d/core/components/CCToggleGroup.js [hunk 1]
```diff
@@ -127,4 +115,14 @@ var ToggleGroup = cc.Class({
 
 });
 
+/**
+ * !#en Read only property, return the toggle items array reference managed by toggleGroup.
+ * !#zh 只读属性，返回 toggleGroup 管理的 toggle 数组引用
+ * @property {Array} toggleItems
+ */
+cc.js.get(ToggleGroup.prototype, 'toggleItems', function () {
```
Comment (jareguo) [inline cocos2d/core/components/CCToggleGroup.js]: 需要 require js

File: cocos2d/core/components/CCToggleGroup.js [hunk 2]
```diff
@@ -127,4 +115,14 @@ var ToggleGroup = cc.Class({
 
 });
 
+/**
+ * !#en Read only property, return the toggle items array reference managed by toggleGroup.
+ * !#zh 只读属性，返回 toggleGroup 管理的 toggle 数组引用
+ * @property {Array} toggleItems
+ */
```
Comment (jareguo) [inline cocos2d/core/components/CCToggleGroup.js]: 把  API 文档去掉

File: cocos2d/core/components/CCToggleGroup.js (hunk 1 above)
Comment (knoxHuang) [reply cocos2d/core/components/CCToggleGroup.js]: 需要？不然会导致报错？

**Pull request merged**